doc/splitjoin.txt

```diff
@@ -1433,6 +1450,30 @@ When set to 0, as is the default, it will look like this:
     </div>
 <
 
+                                 *splitjoin_php_method_chain_until_end_of_line*
```

Comment:
What do you think about the name `splitjoin_php_method_chain_full`? I know there's a lot of verbose variable names, but I try to come up with shorter options whenever I can. Do you think "full method chain" is understandable enough?